keyboards/canoe/canoe.h

```diff
@@ -18,10 +18,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef CANOE_H
 #define CANOE_H
 
-#define KNO KC_NO
-
 #include "quantum.h"
 
+#define LAYOUT_ISO( \
```

Comment:
The convention has been to use LAYOUT_lowercase, please change this to `LAYOUT_iso'